types/simple-peer/index.d.ts

```diff
@@ -1,40 +1,77 @@
-        answerConstraints?: {}; // custom answer constraints (used by createAnswer method)
-        sdpTransform?<T extends any>(sdp: T): T; // function to transform the generated SDP signaling data (for advanced users)
-        stream?: MediaStream; // if video/voice is desired, pass stream returned from getUserMedia
-        streams?: MediaStream[]; // an array of MediaStreams returned from getUserMedia
-        trickle?: boolean; // set to false to disable trickle ICE and get a single 'signal' event (slower)
-        wrtc?: {}; // RTCPeerConnection/RTCSessionDescription/RTCIceCandidate
-        objectMode?: boolean; // set to true to create the stream in Object Mode. In this mode, incoming string data is not automatically converted to Buffer objects.
+        /** set to `true` if this is the initiating peer */
+        initiator?: boolean;
+        /** custom webrtc data channel configuration (used by [`createDataChannel`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createDataChannel)) */
+        channelConfig?: RTCDataChannelInit;
+        /** custom webrtc data channel name */
+        channelName?: string;
+        /** custom webrtc configuration (used by [`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) constructor) */
+        config?: RTCConfiguration;
+        /** custom offer options (used by [`createOffer`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createOffer) method) */
+        offerOptions?: RTCOfferOptions;
+        /** custom answer options (used by [`createAnswer`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createAnswer) method) */
+        answerOptions?: RTCAnswerOptions;
+        /** function to transform the generated SDP signaling data (for advanced users) */
+        sdpTransform?: (this: Instance, sdp: string) => string;
+        /** if video/voice is desired, pass stream returned from [`getUserMedia`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) */
+        stream?: MediaStream;
+        /** an array of MediaStreams returned from [`getUserMedia`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) */
+        streams?: MediaStream[];
+        /**  set to `false` to disable [trickle ICE](http://webrtchacks.com/trickle-ice/) and get a single 'signal' event (slower) */
+        trickle?: boolean;
+        /** custom webrtc implementation, mainly useful in node to specify in the [wrtc](https://npmjs.com/package/wrtc) package. */
+        wrtc?: {
+            RTCPeerConnection: typeof RTCPeerConnection;
+            RTCSessionDescription: typeof RTCSessionDescription;
+            RTCIceCandidate: typeof RTCIceCandidate;
+        };
+        /** set to true to create the stream in Object Mode. In this mode, incoming string data is not automatically converted to Buffer objects. */
+        objectMode?: boolean;
```

Comment:
Same here, I don't think this is present in 9.11

Reply:
`objectMode` is always there.
See https://nodejs.org/api/stream.html#stream_object_mode
https://github.com/feross/simple-peer/tree/d972548299a50f836ca91c36e39304ef0f9474b7#peer--new-peeropts
https://github.com/feross/simple-peer/blob/d972548299a50f836ca91c36e39304ef0f9474b7/index.js#L28-L34